app/views/members/edit.haml

```diff
@@ -13,21 +13,21 @@
   = f.text_field :lastname
 
   = f.date_select :date_of_birth,
-    start_year: Date.today.year - 100,
-    end_year: Date.today.year,
+    start_year: Date.today.year - 50,
```

Comment:
nope, we currently have members older than that ;-) either keep 100 or go for 60 ;-)